src/app/components/feed/SaveFeed.js

```diff
@@ -507,7 +530,7 @@ const SaveFeed = (props) => {
         <FeedCollaboration
           collaboratorId={feedTeam?.team_id}
           feed={feed}
-          onChange={setNewInvites}
+          onChange={handleSetNewInvites}
```

Comment:
Same as handleSetDataPoints, I used a handler function to avoid making changes to additional files and convert a single input update to a two input update

Reply:
Sounds good!

Reply:
Sounds good!